cmd/edit/cluster/cmd.go

```diff
@@ -179,6 +183,20 @@ func init() {
 		"The ARN of the role that is used to forward audit logs to AWS CloudWatch.",
 	)
 
+	flags.StringVar(
+		&args.autonode,
+		"autonode",
+		"",
+		"Enable AutoNode for automatic node management.",
```

Comment:
We should inform the user that the only value available is `enabled`, since it's in the value maybe we could reword entirely

`Supply the value 'enabled' to enable AutoNode for automatic node management.`